component_info.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Gents, are we moving these into a subfolder and not the root directory? // @DelusionalOptimist @mgfeller

Reply:
Yes, the utility has a new `-o` flag for specifying output directory. We decided upon a new `helpers` directory. Maybe the makefile should be updated in accordance with this.

Reply:
Lead the way, @DelusionalOptimist 👍